client/test_tooltool.py

```diff
@@ -1316,7 +1302,7 @@ def tearDown(self):
     def mocked_urllib2(self, data, exp_size=4096, exp_token=None):
         with mock.patch(urlopen_module_as_str) as urlopen:
             def fake_read(url, size):
-                eq_(size, exp_size)
+                assert size == exp_size
```

Comment:
Why the mixed unittest / pytest style assertion formats? Personally I like this style better, but if most of the rest of the file is using unittest style, maybe it's better to be consistent?

Reply:
Initially I was going to make them `self.assertEqual`, but then realized some of them were not in a class, or not in a `TestCase` subclass (the `Handler` class in `UploadTests`), so I've ended up with a bit of a mess.  Arguably the fact we already had some `assertEqual` and some `eq_` means there was already a mix, but I'll rework this to use `assertEqual` where possible.  Thanks.